.web/__init__.py

```diff
@@ -204,9 +205,10 @@ def legacy_images(avatar_id, game='paladins', folder='avatar'):
           return redirect(url_for('static', filename=f'{game}/{folder}/{_}', _external=True))
         return send_from_directory(path, _)
 
-  @app.route('/<game>/<folder>/', defaults={'file': None}, methods=['GET'])
-  @app.route('/<game>/<folder>/<file>', methods=['GET'])
-  def cdn_(game, folder, file):
+  @app.route('/<game>/<folder>/', defaults={'file': None, 'format':''}, methods=['GET'])
+  @app.route("/<game>/<folder>/<file>", defaults={'format':''}, methods=['GET'])
+  @app.route("/<game>/<folder>/<file><any('.gif', '.png', '.jpg':format>")
```

Comment:
`ValueError: malformed url rule: "/<game>/<folder>/<file><any('.gif', '.png', '.jpg':format>"` here. Missing a `)`

Also, change to: `"/<game>/<folder>/<file>.<any('gif', 'png', 'jpg'):format>"`

Reply:
oopsies, i am worrying about the `any` portion tho, since i only saw it once on stackoverflow and its not documented anywhere else.  I would only rely on you to verify that it works.